tests/test_odp_config.py

```diff
@@ -3,7 +3,7 @@
 # you may not use this file except in compliance with the License.
 # You may obtain a copy of the License at
 #
-# http:#www.apache.org/licenses/LICENSE-2.0
+# http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
```suggestion
# https://www.apache.org/licenses/LICENSE-2.0
```

Reply:
@andrewleap-optimizely I can't approve the PR i created lol. 
Looks good to me, nothing major stood out, I think well covered (we also went through it a few times).
(Jae may have ats specific requests.)